kahuna/app/views/main.scala.html

```diff
@@ -33,7 +37,11 @@
     <script>
         this._clientConfig = {
           sessionId: "@Html(sessionId)",
-          googleTrackingId: "@Html(googleTrackingId.getOrElse(""))"
+          googleTrackingId: "@Html(googleTrackingId.getOrElse(""))",
+          feedbackForm: "@Html(feedbackForm.getOrElse(""))",
```

Comment:
minor: should these new config keys have a `Link` suffix to match how they are used clientside?

Reply:
Good point, i've changed this now